ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2

```diff
@@ -9,6 +9,9 @@ node.name: "{{ elasticsearch_node_name }}"
 
 transport.tcp.port: {{ elasticsearch_tcp_port }}
 
+# don't expose ES to public
+network.bind_host: localhost
```

Comment:
I don't think this will work on any non-monolith environment since it needs to be open on the private interface. guessing this also needs to be overrideable